padding/padding.go

```diff
@@ -110,23 +122,45 @@ func (w *Writer) pad() error {
 	return nil
 }
 
-// Close will finish the padding operation. Always call it before trying to
-// retrieve the final result.
-func (w *Writer) Close() error {
-	// don't pad empty trailing lines
-	if w.lineLen == 0 {
-		return nil
+// Close will finish the padding operation and then closes the writer.
+// Notice that the writer will never be wrote again after it closed.
```

Comment:
Typo/grammar: "Notice that the writer can never be written to again once it has been closed."

Reply:
Thanks for the correction!